Initiation/ProjectCharter.md

```diff
@@ -2,30 +2,46 @@
 # Project Charter
 
 ## Project Title
-* 
+* **Volta Card**
 ## Project Purpose 
-*
+* View Promotions and Discount 
```

Comment:
Get Promotion for various kinds of Stores e.g.(Restaurants, Fashions Stores,...), and to be able View The promotions and Discounts on the Map using API from Google Maps